locale/en/manageiq.po

```diff
@@ -29989,6 +29989,10 @@ msgstr ""
 msgid "Editing tag mapping from label \"%{name}\""
 msgstr ""
 
+#: ../app/javascript/components/MarkdownPreview/index.jsx
+msgid "Editor"
```

Comment:
looks like the editor tab title from https://github.com/ManageIQ/manageiq-ui-classic/pull/8972